src/main/java/com/synopsys/integration/detect/lifecycle/run/RunManager.java

```diff
@@ -210,6 +242,20 @@ private UniversalToolsResult runUniversalProjectTools(
 
             DetectableToolResult detectableToolResult = detectableTool.execute(directoryManager.getSourceDirectory());
 
+            NameVersion nameVersion = detectableToolResult.getDetectToolProjectInfo()
```

Comment:
Use project name version, this is not the correct name version and this is not a good default.